dbt_project.yml

```diff
@@ -1,16 +1,19 @@
 name: 'pardot_source'
 version: '0.2.0'
```

Comment:
should we update to 0.3.0 bc of the custom schemas?

Reply:
The version was incorrectly set originally. The tag on the release is actually v0.1.0 currently so keeping v0.2.0 takes into account the breaking change from custom schemas.

Reply:
ah sweet